src/modules/aws_account/entity/index.ts

```diff
@@ -2,32 +2,89 @@ import { Column, Entity, PrimaryGeneratedColumn, PrimaryColumn } from 'typeorm';
 
 import { cloudId } from '../../../services/cloud-id';
 
+/**
+ * Table that will hold the user's AWS credentials. When a new connection to IaSQL is issued,
+ * the AWS credentials are stored.
+ * The keys can be generated from the AWS console for each registered user
+ *
+ * @example
+ * ```sql
+ *  INSERT INTO aws_credentials (access_key_id, secret_access_key)
+ *  VALUES ('${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}');
```

Comment:
this is technically specific to node.js so we should do something like `axdfa***`